packages/validator/src/commands/convertCmd.ts

```diff
@@ -41,9 +67,9 @@ if (process.env.npm_config_template) {
       res = tmp;
     }
   } else {
-    messages.push(invalidConversionMessage(from, to));
+    messages.push(invalidConversionMessage(args['from'] as unknown as string, args['to'] as unknown as string));
```

Comment:
Why do you need to force those types? Both `args.from` and `args.to` are of type `string`.

Reply:
I had some issues with types, which I resolved, but forgot to update this line, updated in da523b323077b3a7c9a8b396bfd8640243d2c9d1